### PR TITLE
chore: remove blueprint from user activity page

### PR DESCRIPTION
**Pull request by magnew**
### Description:

Removes blueprint from user activities page. ~~This page uses a lot of styled components, which we could also update, but for now I focussed on blueprint.~~

Also removes styled components.  

### Reviewer actions

- [ ] I have manually tested the changes in the preview environment
- [ ] I have reviewed the code
- [ ] I understand that "request changes" will block this PR from merging

### CODE REVIEW

File: packages/frontend/src/pages/UserActivity.styles.ts [hunk 1]
```diff
@@ -20,7 +19,11 @@ export const Container = styled.div`
         'table-dashboard-views table-dashboard-views table-chart-views table-chart-views';
 `;
 
-export const ActivityCard = styled(Card)<{ grid: string }>`
+export const ActivityCard = styled.div<{ grid: string }>`
+    background-color: #fff;
+    border-radius: 2px;
+    padding: 20px;
+    border: 1px solid #dee2e6;
```
Comment (magnew) [inline packages/frontend/src/pages/UserActivity.styles.ts:26]: Styles that used to come from the card

File: packages/frontend/src/pages/UserActivity.styles.ts [hunk 2]
```diff
@@ -44,12 +47,12 @@ export const BigNumber = styled.div`
     font-size: 4em;
     line-height: 1.196em;
     font-weight: 500;
-    color: ${Colors.DARK_GRAY4};
+    color: #383e47;
```
Comment (magnew) [inline packages/frontend/src/pages/UserActivity.styles.ts:50]: From blueprint colors. I figured it was ok to hardcode these for now.

File: packages/frontend/src/pages/UserActivity.styles.ts (hunk 1 above)
Comment (IrakliJani) [reply packages/frontend/src/pages/UserActivity.styles.ts:26]: we can use mantine's Card component here. but if its too much work then ignore

File: packages/frontend/src/pages/UserActivity.styles.ts (hunk 2 above)
Comment (IrakliJani) [reply packages/frontend/src/pages/UserActivity.styles.ts:50]: can we use mantine equivalent of it?

File: packages/frontend/src/pages/UserActivity.styles.ts [hunk 3]
```diff
@@ -44,12 +47,12 @@ export const BigNumber = styled.div`
...
 `;
 
 export const BigNumberLabel = styled.h2`
     text-align: center;
-    color: ${Colors.GRAY3};
```
Comment (IrakliJani) [inline packages/frontend/src/pages/UserActivity.styles.ts:52]: same here

**Review by IrakliJani — APPROVED** [review]
looks good to me !

there is one minor one but I think this issue already existed in prod.
CleanShot 2023-12-20 at 16 34 56@2x

**Comment by magnew** [comment]
> looks good to me !
> 
> there is one minor one but I think this issue already existed in prod. CleanShot 2023-12-20 at 16 34 56@2x

Yea, that could be improved. I didn't change it though.

**Pull request merged**